doc/sphinx/practical-tools/utilities.rst

```diff
@@ -268,13 +268,12 @@ file timing data:
 
 
 + ``print-pretty-timed-diff``
-    this target builds a table of timing
-    changes between two compilations; run ``make make-pretty-timed-before`` to
-    build the log of the “before” times, and run ``make make-pretty-timed-
-    after`` to build the log of the “after” times. The table is printed on
-    the command line, and stored in ``time-of-build-both.log``. This target is
-    most useful for profiling the difference between two commits to a
-    repo.
+    this target builds a table of timing changes between two compilations; run
+    ``make make-pretty-timed-before`` to build the log of the “before” times,
+    and run ``make make-pretty-timed-after`` to build the log of the “after”
+    times. The table is printed on the command line, and stored in
+    ``time-of-build-both.log``. This target is most useful for profiling the
+    difference between two commits to a repo.
```

Comment:
"to a repo" or "in a repo"?
Also, shouldn't we avoid such colloquial abbreviations and say "repository" instead?

Reply:
Oh, this should definitely be "in a repo".  I plan to push a fix when I get to a computer today.  No preference on repo vs repository, but I can change it according to your preference.

Reply:
Let's go for "in a repository" then.

Reply:
Done.